sheetsu/core/core.py

```diff
@@ -19,7 +20,7 @@ class Resource(object):
     def __init__(self, client):
         """Initialize Sheetsu Resource variables
         :param client: "SheetsuClient" object instance"""
-        self.sheetsu_api_url = "https://sheetsu.com/apis/v1.0/"
+        self.sheetsu_api_url = "https://sheetsu.com/apis/v1.0oy/"
```

Comment:
Hey @jdrzj , can you tell me why the **oy** in the end of the url?

Reply:
*oy* it's our way to know how users are using API. *o* is for language libs, and *y* is for python.

Reply:
Ah, okay. Cool!